resources/views/livewire/ticket/ticket.blade.php

```diff
@@ -109,14 +129,16 @@
                                 <x-label>
                                     {{ __('Author') }}
                                 </x-label>
+                                <x-toggle :left-label=" __('User') " :label=" __('Contact') " wire:model.live="authorType" />
```

Comment:
also not necessary to be live i guess

Reply:
This needs to be live, because there is a updatedAuthorType() function which updates authenticatable_type and authenticatable_id